### PR TITLE
[feature] init mono repo with two services discussing using grpc

### LINKED ISSUE #8: Find a way to share config (eslint, prettier, ...) across microservices

### PR DESCRIPTION
**Pull request by resanchez**
Fix #8

### CODE REVIEW

**Comment by Totoketchup** [comment]
Push the lock you get in api-gateway

**Comment by Totoketchup** [comment]
Add newlines in all your files as well ;)

File: src/services/api-gateway/apps/api-gateway/src/grpc-database-client.options.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Transport, ClientOptions } from '@nestjs/microservices';
+import { join } from 'path';
+
+export const grpcDatabaseClientOptions: ClientOptions = {
+  transport: Transport.GRPC,
+  options: {
+    package: 'database',
+    protoPath: join(__dirname, '../../../node_modules/interfaces/database/database.proto'),
```
Comment (Totoketchup) [inline src/services/api-gateway/apps/api-gateway/src/grpc-database-client.options.ts:8]: Do you think there is a way avoiding loading them like this ? But from the lib like an 'import proto from "interfaces/database'.

**Comment by Totoketchup** [comment]
Screenshot 2020-04-02 at 19 00 17
The tests are failing

File: src/services/api-gateway/apps/api-gateway/src/grpc-database-client.options.ts (hunk 1 above)
Comment (resanchez) [reply src/services/api-gateway/apps/api-gateway/src/grpc-database-client.options.ts:8]: Yes i don't really like this either, but just checked and Nest only accept a protoPath as string in options :/

**Comment by resanchez** [comment]
I dropped the e2e tests for now as they don't really make sense since we have nothing to test (and I didn't manage to make them work with microservice archi...).

**Pull request merged**